README.md

```diff
@@ -81,82 +81,40 @@ Please note that this package is still under active development and the API is s
-
-### `config.data`
-
-Data used to hydrate the Utility Connect front-end and interface with the API. The format of the `data` object is dependent on the user flow specified by the `scope` parameter.
-
-#### `config.data` for `scope: 'create'`:
-
-Example `config.data` when in the "creating user" flow:
+Example `config.newCredentialData` when in the "creating user" flow:
 
 ```javascript
 {
-  user: {
-    email: 'fake_email@example.com',
-    firstName: 'First',
-    lastName: 'Last',
-    zipCode: '20009',
-  }
+  zip_code: '11787';
 }
 ```
 
 Data is expected in the following format for the `create` scope:
 
-| Data field       | Description                                             | Required |
-| ---------------- | ------------------------------------------------------- | -------- |
-| `user`           | Object specifying the User submitting their credentials | yes      |
-| `user.email`     | user's email                                            | yes      |
-| `user.firstName` | user's first name                                       | yes      |
-| `user.lastName`  | user's last name                                        | yes      |
-| `user.zipCode`   | user's zip code                                         | no       |
-
-#### `config.data` for `scope: 'update'`
-
-Example `config.data` when in the "updating user" flow:
-
-```javascript
-{
-  user: {
-    id: 1, // user's id in the API
-  },
-  utilityCredential: {
-    id: 2 // utility credential id in the API
-  }
-}
-```
-
-Data is expected in the following format for the `update` scope:
-
-| Data Field              | Description                               | Required |
-| ----------------------- | ----------------------------------------- | -------- |
-| `user.id `              | User ID for user being updated            | yes      |
-| `utilityCredential.id ` | User's UtilityCredential ID being updated | yes      |
+| Data field | Description     | Required |
+| ---------- | --------------- | -------- |
+| `zip_code` | user's zip code | no       |
 
 ### `config.accessToken`
 
 An OAuth access token to authenticate API requests. See [Creating OAuth Tokens](https://developers.arcadia.com/#operation/createOAuthToken).
 
-Note that the type of `accessToken` to be used depends on the user flow selected with the `scope` config param:
+Note that the type of `accessToken` to be used depends on the user flow:
 
-- For `scope: 'create'` configs, the `accessToken` must have been created with `scope: 'utility_connect', grant_type: 'client_credentials'`
-- For `scope: 'update'` configs, the `accessToken` must have been created with `scope: 'utility_connect', grant_type: 'password', user_id: xxx`
+- For `mode: 'create'`, the `accessToken` must have been created with `scope: 'utility_connect', grant_type: 'client_credentials'`
+- For `mode: 'update'`, the `accessToken` must have been created with `scope: 'utility_connect', grant_type: 'password', user_id: xxx`
```

Comment:
Same comments as above, I think we should rename to `config.utilityConnectToken`. Also should remove the language around creating with scope or grant_type.